src/content/self-hosted/manage/crds.mdx

```diff
@@ -99,3 +99,49 @@ spec:
 ```
 
 To apply the changes in a manual sceario, you can simply run the command `kubectl apply -f my-catalog-item.yaml` from your terminal to add the catalog item to the your Catalog in the cluster.
+
+## Cloud Credentials
```

Comment:
Same here, should we document it separatly or wrap them under cloud creds?

Reply:
I think centralized here is great